CONTRIBUTING.md

```diff
@@ -24,6 +24,25 @@ Along with this, you will need to have a [MySQL Server](https://dev.mysql.com/do
 
 And an [Elastic Search](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html) server setup
 
+## Remote Environment (Codespaces / Gitpod)
+
+There are shortcut buttons in the README to jumpstart running from a remote environment. When running from these remote environments, consider the following:
+
+- They require at least 8 GB of ram to run the databases
+- If you have a slow remote machine, then the tests may fail when you run them locally if you run them all at the same time from the root directory.
+  - Consider running the tests on at a time or configuring the DEFAULT_DELAY_IN_MS environment variable
+- The work you're doing may not need any databases to be running (i.e UI work)
+  - In this case, consider creating a smaller machine and just running the package you're developing in
+
+### Gitpod
+
+- You must start the databases using the command docker-compose up
```

Comment:
We should either make the following bullet point more clear

> The work you're doing may not need any databases to be running (i.e UI work)

Or reiterate that this is only required if you are doing non ui work.